CONTRIBUTING.md

```diff
@@ -5,22 +5,27 @@ your contribution to Apple and the community, and agree by submitting the patch
 that your contributions are licensed under the Apache 2.0 license (see
 `LICENSE.txt`).
 
-## How to submit a bug report
+## How to Contribute
 
-Please ensure to specify the following:
+Contributions to this project are welcome. You can help by opening an issue to report a bug or suggest a feature, or by submitting a pull request with code changes.
 
-* swift-prometheus commit hash
-* Contextual information (e.g. what you were trying to achieve with swift-prometheus)
-* Simplest possible steps to reproduce
-  * More complex the steps are, lower the priority will be.
-  * A pull request with failing test case is preferred, but it's just fine to paste the test case into the issue description.
-* Anything that might be relevant in your opinion, such as:
-  * Swift version or the output of `swift --version`
-  * OS version and the output of `uname -a`
-  * Network configuration
+For security concerns, please follow the private disclosure process outlined in the [SECURITY.md](SECURITY.md) file instead of opening a public issue.
 
+### Reporting Bugs or Requesting Features
 
-### Example
+A great bug report or feature request is specific and actionable. Before submitting a new issue, please check if a similar one already exists.
+
+When you create a bug report, please provide the following:
+
+- **swift-prometheus commit hash** you are using.
```

Comment:
All great detail, and we should include a PR template that specifically asks for this detail as well

Reply:
I was thinking about it. Let’s keep track of creating a more formal template in /swiftlang or /apple?

(We are currently exploring the possibility of having this client accepted into the Prometheus project. Consequently, the Contributing guidance may undergo further modifications if this outcome materializes.)